src/components/call-tool.js

```diff
@@ -131,7 +131,19 @@ module.exports = React.createClass({
       <section>
         <h2 className="bold">{this.context.intl.formatMessage({id: 'calling_headline'})}</h2>
         <div>{this.context.intl.formatMessage({id: 'calling_tagline'})}</div>
-        <div className="bold">SOCIAL ICONS GO HERE</div>
+        <div className="social-container">
+          <i className="fa fa-facebook" aria-hidden="true"></i>
```

Comment:
Small JSX note: it's an XML syntax, not technically HTML, so any element without content can be self-closed: `<i className="fa fa-facebook" aria-hidden="true" />`, making it a tiny bit nicer.